### DISCUSSION TITLE
When will the project move from Beta ?

### DISCUSSION THREAD

**Discussion by whiletruelearn** [ourownstory/neural_prophet#389] [General]
Hi,

This is more an enquiry on the project status. I see that currently project released in beta. I have been using the library on and off for couple of months now. A colleague pointed me to the fact that releases are still made as in beta. I haven't found any rough edges so far with the library, but wanted to get a sense from the author itself on the stability of the library and when it will probably come out of beta status. 

Thanks again for building such a fantastic library.

**Comment by ourownstory** [comment]
Hi @whiletruelearn Thank you for the feedback and for being a part of the community!

Your colleague is not wrong, but we actually do intend remove the beta label soon. It will be more of a soft transition, as we have already made many improvements since the initial beta release.
You can expect it to happen by  the end of the year, and we will also release a scientific writeup to help understand the model components and performance.
